schema/bidder_position.js

```diff
@@ -31,25 +33,20 @@ const BidderPositionType = new GraphQLObjectType({
     },
     is_winning: {
       type: GraphQLBoolean,
-      resolve: (position) => {
-        return gravity(`sale_artwork/${position.sale_artwork_id}`)
-          .then((saleArtwork) => {
-            return saleArtwork.highest_bid.id === position.highest_bid.id;
-          });
-      },
-    },
-    display_max_bid_amount_dollars: {
-      type: GraphQLString,
-    },
-    display_suggested_next_bid_dollars: {
-      type: GraphQLString,
-    },
-    max_bid_amount_cents: {
-      type: GraphQLInt,
-    },
-    suggested_next_bid_cents: {
-      type: GraphQLInt,
+      resolve: (position) =>
+        gravity(`sale_artwork/${position.sale_artwork_id}`)
+          .then(saleArtwork =>
+            get(saleArtwork, 'highest_bid.id') === get(position, 'highest_bid.id')
```

Comment:
This is the only real thing that's changed, basically just safe object access since sometimes the highest_bid is null